core/modules/user/store/actions.ts

```diff
@@ -257,16 +258,24 @@ const actions: ActionTree<UserState, RootState> = {
       return ordersHistory
     }
   },
-  async refreshOrdersHistory ({ commit }, { resolvedFromCache, pageSize = 20, currentPage = 1 }) {
+  async refreshOrdersHistory ({ commit, getters }, { resolvedFromCache, pageSize = 20, currentPage = 1 }) {
```

Comment:
I can see now that here we have pageSize=20, and inside mixin we have 10. Is `10` properly applied in each place?